### PR TITLE
completed coding challenge

### PR DESCRIPTION
**Pull request by devonhackley**

### CODE REVIEW

**Comment by devonhackley** [comment]
- [x ] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [ x] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: code-challenges/401challenges/src/test/java/challenges/LinkedListTest.java [hunk 1]
```diff
@@ -65,4 +65,61 @@
         assertEquals(actual, expected);
     }
 
+    @Test public void testLinkedListAppendMethod(){
+        LinkedList libraryForTest = new LinkedList();
+        libraryForTest.insert(10);
```
Comment (evanslaton) [inline code-challenges/401challenges/src/test/java/challenges/LinkedListTest.java]: It would have been better to have only used `.append()` here instead of also using `.insert()`

File: code-challenges/401challenges/src/main/java/challenges/linkedlist/LinkedList.java [hunk 2]
```diff
@@ -69,4 +69,82 @@ public ArrayList print(){
 
         return nodeValues;
     }
+
+    public void append(int data){
+        try
+        {
```
Comment (evanslaton) [inline code-challenges/401challenges/src/main/java/challenges/linkedlist/LinkedList.java]: Opening `{` should be on the same line as the thing they are opening:

`try {`
